Analysis/HiggsTauTau/scripts/run2_control.sh

```diff
@@ -10,7 +10,7 @@ PROD=$1
 YEAR=$2
 ANA=$3
 POSTFIT=$4
-PARAMS=./scripts/"$PROD"_params_$YEAR.dat
+#PARAMS=./scripts/"$PROD"_params_$YEAR.dat
```

Comment:
I see you've now added an SM version of this script - can you revert this to the master copy for use in the MSSM? Unless you added another copy i havent seen yet...